inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/settings/ProjectWorkloadSettingsPanelFactory.java

```diff
@@ -33,7 +33,7 @@
  * {@link WorkloadManagementAutoConfiguration#projectWorkloadSettingsPanelFactory}.
  * </p>
  */
-@Order(300)
+@Order(600)
```

Comment:
@christophscheunemann is there a reason why the order of the tabs in the settings panel is `300` for the workload panel? then I will not change it back here ;)